opensciencegrid/ospool-cm/Dockerfile

```
@@ -1,4 +1,8 @@
-FROM opensciencegrid/software-base:3.6-el7-release
+# Specify the opensciencegrid/software-base image tag
+ARG BASE_OSG_SERIES=3.5
```

```suggestion
ARG BASE_OSG_SERIES=3.6
```
since we've been using the 3.6 image